_maps/map_files/IceBoxStation/IceBoxStation.dmm

```diff
@@ -40874,6 +40874,10 @@
 /obj/machinery/camera/directional/south{
 	c_tag = "Service-Botany Bottom 2"
 	},
+/obj/machinery/button/door/directional/south{
+	name = "Botany Shutters Control";
+	id = botany_shutters
```

Comment:
This is incorrect

It needs to be `id = "botany_shutters"`
This is why compile maps and MDB are failing

Reply:
Thanks a ton!

Reply:
Don't forget to mark this as resolved